src/main/java/gui/TimelinesGridPane.java

```diff
@@ -97,10 +60,10 @@ public void removeCameraShotBlock(CameraShotBlock block) {
     private void addPanes() {
         panes = new ArrayList<>();
         int c;
-        for (int i = 0; i < numberOfTimelines; i++) {
+        for (int i = 0; i < getNumberOfHorizontalGrids(); i++) {
             c = 1;
-            for (int j = 0; j < numberOfCounts; j++) {
-                SnappingPane pane = new SnappingPane(j, i, 200, countHeight);
+            for (int j = 0; j < getNumberOfVerticalGrids(); j++) {
+                SnappingPane pane = new SnappingPane(j, i, 200, getVerticalElementSize());
```

Comment:
Turns out, the snappingpane class doesn't even need width and height, so I just removed them completely.